pennylane/_qubit_device.py

```diff
@@ -669,8 +673,9 @@ def active_wires(operators):
 
         return Wires.all_wires(list_of_wires)
 
+    # pylint: disable=too-many-statements
     def statistics(
-        self, circuit: QuantumScript = None, shot_range=None, bin_size=None, observables=None
+        self, observables=None, shot_range=None, bin_size=None, circuit: QuantumScript = None
```

Comment:
Changed order to avoid an error when calling:
```python
dev.statistics([], circuit=circuit)
```

Reply:
The statistics method had the following signature in the last release:
```python
def statistics(self, observables, shot_range=None, bin_size=None, circuit=None)
```

In this release I deprecated the use of `observables` to move towards using `circuit` (given that we want to use `circuit.measurements`. We can still access observables using `circuit.observables`). This PR is moving `observables` back to the beginning, such that there is no breaking change with the added deprecation cycle.